test/kpod_build.bats

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "bud-from-scratch" {
+  command -v buildah >/dev/null 2>&1 || skip
```

Comment:
@rhatdan I thought this line would make the test be skipped if Buildah wasnt't available.  hoewver it seems to be moving right into the tests. Do you have other syntax that I might try?  I  first went with an exit to the right of the pipes but that didn't work so I tried skip instead.

Reply:
How about
```
which buildah >/dev/null 2>&1 || skip
```

Reply:
This works for SELinux.  Might need to add some info for the skip line?
```
    if [ ! -e /usr/sbin/selinuxenabled ] || [ ! /usr/sbin/selinuxenabled ]; then
        skip "SELinux not enabled"
    fi
```

Reply:
@rhatdan thanks.  I was hoping to avoid which, but it looks like we're using it in other places for git and runc, think I'll convert to that.